app/views/application_form_mailer/_project_info.slim

```diff
@@ -0,0 +1,26 @@
+- application = local_assigns.fetch(:application)
+- data        = application.data
+- project1    = application.project1
+- project2    = application.project2
+
+h3 About your project
+
+b = Application.data_label('project1_id')
+p = link_to_if project1, project1&.name, project1
+
+b = Application.data_label('why_selected_project1')
+= simple_format(auto_link(show_or_na(data.why_selected_project1)))
+
+b = Application.data_label('plan_project1')
+= simple_format(show_or_na(render_markdown(data.plan_project1)))
```

Comment:
I feel at some point we should introduce a presenter 🙃   Let's not bother us with it in this PR, though